app-modules/ai/src/Filament/Resources/AiAssistantResource/Pages/EditAiAssistant.php

```diff
@@ -106,6 +106,17 @@ public function getSaveFormAction(): Action
             });
     }
 
+    public static function canAccess(array $parameters = []): bool
+    {
+        $assistant = $parameters['record'];
+
+        if ($assistant->is_default == true) {
```

Comment:
Because this attribute is a bool you don't need to do the check here.

```suggestion
        if ($assistant->is_default) {
```